addons/website_slides/static/src/xml/slide_quiz_create.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+
+    <t t-name="slide.quiz.question.input">
+        <div t-attf-class="o_wsildes_quiz_question_input mt-3 #{!widget.update ? 'col' : ''}" t-att-data-id="widget.question.id || ''">
+            <form class="mb-3">
+                <div class="o_wslides_quiz_question row align-items-center mr-0 mb-2">
+                    <small class="text-muted col-1 text-center">
+                        <span class="h4 o_wslides_quiz_question_sequence"><t t-esc="widget.sequence"/></span>.
+                    </small>
+                    <input type="text" name="question-name" class="form-control col-11" placeholder="Enter your question"
+                           t-att-value="widget.question.text" />
+                </div>
+                <div class="text-muted mb-2">
+                    <span>Select the correct answer below :</span>
+                </div>
+                <div class="list-group">
+                    <t t-if="widget.question.answers" >
+                        <t t-foreach="widget.question.answers" t-as="answer" >
+                            <t t-call="slide.quiz.answer.line"/>
+                        </t>
+                    </t>
+                    <t t-else="" >
+                        <t t-foreach="[1, 2, 3]">
+                            <t t-call="slide.quiz.answer.line" />
+                        </t>
+                    </t>
+                </div>
+                <div class="alert alert-danger o_wslides_js_quiz_create_error mt-2 d-none" role="alert">
+                    <span></span>
+                    <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+                        <span>&amp;times;</span>
+                    </button>
+                </div>
```

Comment:
Do you still need that section since you changed the way errors are displayed?

Reply:
I deleted it and changed the display error method to notification as discussed.
I also created a template for the Add Quiz and Add Question buttons